AdminUi/src/AdminUi/ClientApp/src/app/components/client/client-list/client-list.component.ts

```diff
@@ -97,7 +97,7 @@ export class ClientListComponent {
         this.clients = this.serverClients;
         if (this.filter.clientId) this.clients = this.clients.filter((c) => c.clientId.toUpperCase().includes(this.filter.clientId!.toUpperCase()));
         if (this.filter.displayName) this.clients = this.clients.filter((c) => c.displayName.toUpperCase().includes(this.filter.displayName!.toUpperCase()));
-        if (this.filter.tiers && this.filter.tiers.length > 0) this.clients = this.clients.filter((c) => this.filter.tiers!.includes(c.defaultTier));
+        if (this.filter.tiers && this.filter.tiers.length > 0) this.clients = this.clients.filter((c) => this.filter.tiers!.includes(c.defaultTier.id));
```

Comment:
Am I getting this right: the user has to type the ID of the client? That's not good. How should he know the ID? He only sees the name in the UI.

Reply:
As far as I am aware, the Tiers filter currently uses a dropdown without search:
![image](https://github.com/nmshd/backbone/assets/118175875/296d87ec-9589-4bcf-9c94-77b753ec0bd4)
So the user doesn't interact with the IDs directly, only with the Tier names.

Reply:
Ah, sure.